modules/ossm-operatorhub-remove.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/removing-ossm.adoc
+
+[id="ossm-operatorhub-remove_{context}"]
+= Removing the installed Operators
+
+You must remove the Operators to successfully remove {ProductName}. Once you remove the {ProductName} Operator, you must remove the Elasticsearch Operator, Jaeger Operator, and Kiali Operator.
+
+[id="ossm-remove-operator-servicemesh_{context}"]
+== Removing the {ProductName} Operator
+Follow this procedure to remove the {ProductName} Operator.
+
+.Prerequisites
+* Access to the {product-title} web console.
+* The {ProductName} Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalogs* -> *Operator Management*.
+
+. Click the *Project* menu, and then click *openshift-operators*.
+
+. Click the *servicemeshoperator* Options menu {kebab}, and then click *Remove Subscription*.
+
+. Select *Also completely remove the Operator from the selected namespace* and click *Remove*.
+
+[id="ossm-remove-operator-elasticsearch_{context}"]
+== Removing the Elasticsearch Operator
```

Comment:
@knrc @objectiser 
Thinking this should be the last operator removed as Jaeger uses this.

Reply:
I updated this and updated the intro as well to reflect the removal order change.